Source/processcontainers/CMakeLists.txt

```diff
@@ -96,6 +100,49 @@ elseif (PROCESSCONTAINERS_CRUN)
                 PRIVATE
                 libcrun::libcrun
         )
+elseif (PROCESSCONTAINERS_DOBBY)
+        find_package(jsoncpp REQUIRED)
+        find_package(Dobby REQUIRED CONFIG)
+
+        # We need libsystemd since we're using their dbus library (sd-bus)
+        # There's probably a better way of doing this...
+        find_path(SYSTEMD_INCLUDE_DIRS
+                NAMES systemd/sd-bus.h
+        )
+
+        find_library(SYSTEMD_LIBRARIES
+                NAMES systemd
+        )
+
+        add_definitions( -DRDK )
+
+        include(FindPackageHandleStandardArgs)
+        find_package_handle_standard_args(
+                LIBSYSTEMD
+                SYSTEMD_LIBRARIES SYSTEMD_INCLUDE_DIRS
+        )
+        target_link_libraries(${TARGET}
+                PRIVATE
+                # Dobby libraries
+                DobbyClientLib
+                IpcService
+                AppInfraCommon
```

Comment:
WHy is the dependency on AppInfrCommon Needed ?

Reply:
DobbyClientLibIt is not a blocking issue. It is bad practise :-)
Components we develop are self containing so the user of the library does not have to dig around to find potential other dependencDobbyClientLibies he should include.
Dependencies could be managed through pkgconfig files than if in the future the dependency list changes for the  DobbyClientLib, you just change the pkconfig file that you generate with the DobbyClientLib artifact. Than this CMakeList.txt file does *not* require changes, as it just picks up the pkgconfig.
Now, with this setup, if you change the dependency list of the DobbyClientLib (I assume that is the library found through find_package(Dobby REQUIRED CONFIG)) you have to change projects that are just "using" this library. I would expect that to be unnescary.